### ISSUE TITLE
Axis labels are visually too dense

### ISSUE THREAD

**Issue by BANOnotIT** [Devographics/Monorepo#474]
Too many visually different kinds of things in one column. Too distracting from the actual results. Also there are Baseline indicators somewhere.

![image](https://github.com/user-attachments/assets/ed815f79-80d7-44f9-af68-6890ce07cca5)

All things combined provide too many variations that mix together, bringing up this visual clutter.

**Comment by SachaG** [comment]
That's true, but I think the extra functionality provided is worth the clutter.
